docs/en/index.md

```diff
@@ -6,109 +6,109 @@
 
 ## What are you looking for?
 
-[![Getting Started](images/GettingStarted.PNG)](#get-started-with-aaw)
+- [Getting Started with AAW](#kubeflow-account)
```

Comment:
Sure! I think it's not necessary since the same values are in the navigation side menu:

![image](https://user-images.githubusercontent.com/8212170/232582395-a3004371-c008-4f13-a370-074151eb4133.png)